src/cli/Show.cpp

```diff
@@ -49,6 +49,13 @@ int Show::execute(QStringList arguments)
                                QObject::tr("Key file of the database."),
                                QObject::tr("path"));
     parser.addOption(keyFile);
+    QCommandLineOption fields(QStringList() << "a"
+                                            << "show-attribute",
```

Comment:
@cyphar what about `-a` and `--attributes` ? I find it redundant to have both the `show` command and `show` in the name of the option.